apps/cowswap-frontend/src/modules/limitOrders/containers/LimitOrdersConfirmModal/index.tsx

```diff
@@ -36,7 +36,14 @@ export interface LimitOrdersConfirmModalProps {
 }
 
 export function LimitOrdersConfirmModal(props: LimitOrdersConfirmModalProps) {
-  const { inputCurrencyInfo, outputCurrencyInfo, tradeContext, priceImpact, recipient } = props
+  const { inputCurrencyInfo, outputCurrencyInfo, tradeContext: tradeContextInitial, priceImpact, recipient } = props
+
+  /**
+   * This is a very important part of the code.
+   * After the confirmation modal opens, the trade context should not be recreated.
+   * In order to prevent this, we use useMemo to keep the trade context the same when the modal was opened.
+   */
+  const tradeContext = useMemo(() => tradeContextInitial, [])
```

Comment:
Why is the linter not complaining about the missing dependency?

Reply:
https://github.com/cowprotocol/cowswap/pull/4484